tests/components/azure_event_hub/conftest.py

```diff
@@ -59,7 +59,10 @@ async def mock_entry_fixture(hass, filter_schema, mock_create_batch, mock_send_b
         utcnow() + timedelta(seconds=entry.options[CONF_SEND_INTERVAL]),
     )
     await hass.async_block_till_done()
-    return entry
+
+    yield entry
+
+    await entry.async_unload(hass)
```

Comment:
`mock_send_batch` is required during the unload process:

https://github.com/home-assistant/core/blob/3c8397a7b9e863a8c7925865c766054627ab07c7/homeassistant/components/azure_event_hub/__init__.py#L165-L173